src/WhoReacted/index.jsx

```diff
@@ -24,14 +24,18 @@ export default class WhoReacted extends Plugin {
 
         this.defaultSettings = {
             maxUsersShown: 6,
+            imageSize: 20,
             reactionThreshold: 10,
             userThreshold: 100,
-            useHighestUserCount: true
+            useHighestUserCount: true,
+            showBot: false,
```

Comment:
Shouldn't `bots` be in plural?

Reply:
Was meant to be short for "showBotReactors", but makes sense.